aten/src/ATen/templates/StorageDerived.cpp

```diff
@@ -26,6 +27,25 @@ static THCDeviceAllocator storage_deleter = {
   nullptr,
   nullptr,
 };
+static cudaError_t wrapped_alloc(void * ctx, void** result, size_t size, cudaStream_t stream) {
```

Comment:
you don't use stream?

Reply:
If an allocator uses the stream it can access it via `at::globalContext().getCurrentCUDAStream()`. This keeps the Allocator interface simple and consistent between CPU and CUDA.